.github/workflows/test-and-check.yml

```diff
@@ -16,7 +16,7 @@ jobs:
     runs-on: ${{ matrix.os }}
     strategy:
       matrix:
-        python-version: [3.8, 3.9]
+        python-version: [3.9]
```

Comment:
3.8 is a) old and b) breaking the build on OSX-latest.  Removing it seems like the most painless way to handle the issue.